doc/api/deprecations.md

```diff
@@ -925,7 +923,7 @@ This was never a documented feature.
 
 Type: End-of-Life
 
-The `--with-lttng` compile time option is removed.
+The `--with-lttng` option has been removed.
```

Comment:
Nit: I think "compile" was useful here.

Reply:
OK, put `compile-time` in there. Thanks.